src/Data.js

```diff
@@ -75,7 +75,9 @@ export default function Data({ shx }) {
   // +-------------+
 
   const renderError = (reasons) => {
-	return(<div>{Array.isArray(reasons) ? reasons.join('; ') : reasons}</div>);
+       let displayReasons = Array.isArray(reasons) ? reasons : [reasons];
+       displayReasons = displayReasons.map(reason => reason.replace(/^Error: /, ''));
```

Comment:
this needs to be replaceAll or use the g flag, right? otherwise it will only replace the first instance.